### ISSUE TITLE
v1.10RC2 Time is off by 1 hours

### ISSUE THREAD

**Issue by ARiber** [osTicket/osTicket#2495]
Hi,
I noticed it is no more possible to set the daylight saving time in Version 1.10 ?
Our server is located in Germany. The php.ini has the correct timezone Europe/Berlin and by using "Date" the server also shows the correct date. If I check the database entry on a thread it is also correct. But in the ticket is always shows:

Created by XXX "in" xx minutes 

normaly it should be xx minutes ago.

One our later it will be xx minutes ago. 

In the Settings and also the user profile the timezone is set correctly.
In settings the time zone is "Europe/Berlin" and in user profile it is "Asia/HongKong" since I am currently there. 

Any idea what could be the reason?

**Comment by mfelber** [comment]
Do you use php 5.6? This seems to be a duplicate issue. See here: https://github.com/osTicket/osTicket-1.8/issues/2440

**Comment by balojs** [comment]
+1

**Comment by greezybacon** [comment]
@ARiber could you post your system information page?

**Comment by ARiber** [comment]
Hi, actually I had solved this problem a longer time ago. But to be honest I don't exactly remember since it is too long ago. I found the solution in some other thread.

As requested, here is my system info

osTicket-Version	v1.10-rc.2 (231f11e)  Upgrade — v1.10-rc.3 is available
Server-Software	Apache/2.4.10 (Debian)
MySQL-Version	5.5.47
PHP-Version	5.6.17-0+deb8u1
PHP-Erweiterungen
gdlib	 Wird für die Bildmanipulation und PDF-Druck verwendet
imap	 Wird für den E-Mail-Abruf verwendet
xml	 XML API
xml-dom	 Wird für die HTML E-Mail-Verarbeitung verwendet
json	 Verbessert die Leistung beim Erzeugen und Verarbeiten von JSON
mbstring	 Dringend empfohlen für nicht-westeuropäische Sprachinhalte
phar	 Dringend empfohlen für Plug-ins und Sprachpakete
intl	 Dringend empfohlen für nicht-westeuropäische Sprachinhalte
fileinfo	 Wird verwendet, um Dateitypen für Uploads zu erkennen
PHP-Einstellungen
cgi.fix_pathinfo	 "1" wird empfohlen, wenn AJAX nicht funktioniert
date.timezone	 Europe/Berlin
Datenbankinformationen und Nutzung
Schema	ticketsystem (localhost)
Schema-Signatur	98ad7d550c26ac44340350912296e673
Belegter Speicher gesamt	12352.68 MiB
Datenbankspeicher für Anhänge	11044.63 MiB
Timezone	CET (Interpreted as Europe/Berlin)
